bluemira/codes/process/_inputs.py

```diff
@@ -315,7 +716,7 @@ def to_invariable(self) -> Dict[str, _INVariable]:
         """
         out_dict = {}
         for name, value in self:
-            if name not in ["icc", "ixc", "bounds"]:
+            if name not in ["icc", "ixc", "bounds"] and value is not None:
                 new_val = _INVariable(name, value, "Parameter", "", "")
                 out_dict[name] = new_val
         out_dict["icc"] = _INVariable(
```

Comment:
might need to do something similar with icc ixc and bounds if they're None. Not important right now but jsut flagging